data/remnant/remnant missions.txt

```diff
@@ -940,6 +940,143 @@ mission "Remnant: Learn Sign Follow Up"
+	minor
+	source
+		government Remnant
+	destination Bounty
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "Terminus exploration: done"
+		not "Deep: Wormhole Research: done"
+		not "Remnant: Follow-up on Terminus Researchers B: done"
+		not "Remnant: Follow-up on Terminus Researchers B: active"
+		random < 40
+	on offer
+		conversation
+			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`
+			branch secrets
+				has "deep: did not reveal remnant"
+			`	Thinking back, it suddenly occurs to you that you helped retrieve a science drone for some researchers on Bounty - researchers who were studying the wormhole in Terminus. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes. It would only be a matter of time until someone figures out how to use the key stones - either by luck or by research, and then the Remnant's secrecy would vanish.`
+				goto terminuschoice
+			label secrets
+			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`
+			label terminuschoice
+			choice
+				`	(Find a prefect.)`
+				`	(Forget about it.)`
+					decline
+			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
+			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?"`
+			`	He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`
+			branch confirmation
+				has "deep: did not reveal remnant"
+			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
```

Comment:
```suggestion
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
```